icu4j/maven-build/maven-icu4j-charset/pom.xml

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0"?>
+<!--
+* © 2016 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html
+*******************************************************************************
+* Copyright (C) 2010-2016, International Business Machines Corporation and    *
+* others. All Rights Reserved.                                                *
+*******************************************************************************
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  
+  <parent>
+    <groupId>com.ibm.icu</groupId>
+    <artifactId>icu4j-root</artifactId>
+    <version>73.1-SNAPSHOT</version>
+    <!-- default relativePath for parent -->
+  </parent>
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.ibm.icu</groupId>
```

Comment:
Nitpick: also common in maven: leave out `groupId` and `version` if they are the same as the parent.
This is style, so somewhat preferential.

But in my opinion reduces the clutter, makes it clear that the group / version is the same, and reduces the number of places where one has to update the version when we move to ICU.next